index.html

```diff
@@ -1,27 +1,30 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Portfolio</title>
     <link rel="stylesheet" href="styles.css">
 </head>
+
 <body>
     <header>
         <div class="header-img"></div>
         <nav>
-            <button class="mobile-menu"><img src="images/ic_menu.svg" alt="Open Navigation"></button>
+            <button type="button" class="mobile-menu"><img src="images/ic_menu.svg" alt="Open Navigation"></button>
```

Comment:
- [ ] The form is not respinding to click on button because button  type should be `submit`. Please kindly change the button type to `submit` to make the form work ☀️